src/args.jl

```diff
@@ -363,104 +363,115 @@ const _plot_defaults = KW(
-    :extra_kwargs                    => Dict()
+    :title                      => "",
+    :titlelocation              => :center,           # also :left or :right
+    :fontfamily_subplot         => :match,
+    :titlefontfamily            => :match,
+    :titlefontsize              => 14,
+    :titlefonthalign            => :hcenter,
+    :titlefontvalign            => :vcenter,
+    :titlefontrotation          => 0.0,
+    :titlefontcolor             => :match,
+    :background_color_subplot   => :match,            # default for other bg colors... match takes plot default
+    :background_color_legend    => :match,            # background of legend
+    :background_color_inside    => :match,            # background inside grid
+    :foreground_color_subplot   => :match,            # default for other fg colors... match takes plot default
+    :foreground_color_legend    => :match,            # foreground of legend
+    :foreground_color_title     => :match,            # title color
+    :color_palette              => :auto,
+    :legend                     => :best,
+    :legendtitle                => nothing,
+    :colorbar                   => :legend,
+    :clims                      => :auto,
+    :colorbar_ticks             => :auto,
+    :colorbar_tickfontfamily    => :match,
+    :colorbar_tickfontsize      => 8,
+    :colorbar_tickfonthalign    => :hcenter,
+    :colorbar_tickfontvalign    => :vcenter,
+    :colorbar_tickfontrotation  => 0.0,
+    :colorbar_tickfontcolor     => :match,
+    :colorbar_scale             => :identity,
+    :colorbar_formatter         => :auto,
+    :colorbar_discrete_values   => [],
+    :colorbar_continuous_values   => zeros(0),
+    :legendfontfamily           => :match,
+    :legendfontsize             => 8,
+    :legendfonthalign           => :hcenter,
+    :legendfontvalign           => :vcenter,
+    :legendfontrotation         => 0.0,
+    :legendfontcolor            => :match,
+    :legendtitlefontfamily      => :match,
+    :legendtitlefontsize        => 11,
+    :legendtitlefonthalign      => :hcenter,
+    :legendtitlefontvalign      => :vcenter,
+    :legendtitlefontrotation    => 0.0,
+    :legendtitlefontcolor       => :match,
+    :annotations                => [],                # annotation tuples... list of (x,y,annotation)
+    :projection                 => :none,             # can also be :polar or :3d
+    :aspect_ratio               => :auto,             # choose from :none or :equal
+    :margin                     => 1mm,
+    :left_margin                => :match,
+    :top_margin                 => :match,
+    :right_margin               => :match,
+    :bottom_margin              => :match,
+    :subplot_index              => -1,
+    :colorbar_title             => "",
+    :colorbar_titlefontsize     => 10,
+    :colorbar_title_location    => :center,           # also :left or :right
+    :colorbar_fontfamily        => :match,
```

Comment:
Where do I do the matching the colorbar_fontfamily, colorbartitlefont and colorbar_titlefontcolor to the rest of the fonts?

Reply:
I think you are looking for https://github.com/JuliaPlots/Plots.jl/blob/f76390c20d188a35f517a8cf467c061325d1c22e/src/args.jl#L1342-L1344